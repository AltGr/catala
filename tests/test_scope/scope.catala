@Article@

/*
new scope A:
  param a type int
  param b type int
  param c type bool

scope A:
  def c := false
  def a [ c ] := 42
  def a [ not c ] := 0
  def b [ not c ] := 1337
  def b [ not c ] := 0
*/
