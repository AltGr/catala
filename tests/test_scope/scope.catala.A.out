[ERROR] There is a conflict between multiple rules for assigning the same variable.
[ERROR] 
[ERROR] This value is available because this justification is true:
[ERROR]   --> test_scope/scope.catala
[ERROR]    | 
[ERROR] 13 |   def b [ not c ] := 1337
[ERROR]    |           ^^^^^
[ERROR] 
[ERROR] This value is available because this justification is true:
[ERROR]   --> test_scope/scope.catala
[ERROR]    | 
[ERROR] 14 |   def b [ not c ] := 0
[ERROR]    |           ^^^^^
